### ISSUE TITLE
This is working?

### ISSUE THREAD

**Issue by franzejr** [danielweinmann/react-native-keystore#1]

**Comment by danielweinmann** [comment]
No, I just started the project but ended up using https://github.com/jasonmerino/react-native-simple-store instead (which does not do the same, but solved my case).

**Comment by pesehr** [comment]
@danielweinmann @franzejr 
you can use this: https://github.com/pesehr/react-native-secure-KeyStore
